package.json

```diff
@@ -80,5 +80,8 @@
     "packages/*",
     "configs/*",
     "scripts"
-  ]
+  ],
+  "dependencies": {
+    "electron": "^25.9.0"
+  }
```

Comment:
You’ll probably want to undo this change and then run `npm install` again, it should keep the package-lock basically as it was before (except for metadata fields)

Reply:
Done :D